### PR TITLE
Mobj get va

### PR DESCRIPTION
**Pull request by jenswi-linaro**

### CODE REVIEW

File: core/include/mm/mobj.h [hunk 1]
```diff
@@ -156,7 +166,7 @@ static inline void mobj_put(struct mobj *mobj)
  */
 static inline void mobj_put_wipe(struct mobj *mobj)
 {
-	void *buf = mobj_get_va(mobj, 0);
+	void *buf = mobj_get_va(mobj, 0, 1);
```
Comment (etienne-lms) [inline core/include/mm/mobj.h:169]: why not `mobj->size`?

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` for commit "core: fix argument checks in yielding_call_with_arg()".

Aside 1 question, commit "core: make mobj_get_va() more secure" LGTM.

File: core/include/mm/mobj.h (hunk 1 above)
Comment (jenswi-linaro) [reply core/include/mm/mobj.h:169]: That should work too. I'll change it.

**Comment by jenswi-linaro** [comment]
Update

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` for commit "core: make mobj_get_va() more secure".

**Comment by jforissier** [comment]
For "core: fix argument checks in yielding_call_with_arg()":
`Reviewed-by: Jerome Forissier <jerome@forissier.org>`

**Comment by jenswi-linaro** [comment]
Squashed and tags applied.

**Pull request merged**